tests/v2/integration/setup/main.go

```diff
@@ -21,17 +38,20 @@ func main() {
 		Password: "admin",
 	}
 
-	hostURL := "localhost:8443"
+	ipAddress := getOutboundIP()
+	hostURL := fmt.Sprintf("%s:8443", ipAddress.String())
 	token, err := token.GenerateUserToken(user, hostURL)
 	if err != nil {
 		logrus.Fatalf("error with generating admin token: %v", err)
 	}
 
+	clusterName := namegen.AppendRandomString(k3dClusterNameBasename)
+
 	cleanup := true
 	rancherConfig.AdminToken = token.Token
 	rancherConfig.Host = hostURL
 	rancherConfig.Cleanup = &cleanup
-	rancherConfig.ClusterName = "local"
+	rancherConfig.ClusterName = clusterName
```

Comment:
Does this mean that tests will only now be run against the new k3d downstream cluster? What if we still want to run some tests on the local?

Reply:
By default yes. ClusterName should denote the downstream cluster that tests are being ran against. If a local cluster is needed to run tests against then the test should specify that.